libqtile/config.py

```diff
@@ -369,6 +369,8 @@ class Screen(CommandObject):
     resized to fill it. If the mode is ``"stretch"``, the image is stretched to fit all
     of it into the screen.
 
+    The ``x11_drag_polling_rate`` parameter specifies the rate for drag events in the X11 backend. By default this is set to 120, but if you prefer it you can set it lower for better performance or higher if you have a high refresh rate monitor. 120 would mean that we handle a drag event 120 times per second.
```

Comment:
Can you wrap this line and the line in the changelog? Other than that this patch looks awesome, thank you for doing it!

Reply:
Thanks!